src/DataSource/ApiDataSource.php

```diff
@@ -198,4 +198,22 @@ public function sort(Sorting $sorting)
 		return $this;
 	}
 
+	/**
+	 * @param string $aggregation_type
+	 * @param string $column
+	 * @return mixed
+	 */
+	public function addAggregationColumn($aggregation_type, $column)
+	{
+		// TODO: Implement addAggregationColumn() method.
+	}
+
+	/**
+	 * get aggregation row
+	 * @return array
+	 */
+	public function getAggregationData()
+	{
+		// TODO: Implement getAggregationData() method.
```

Comment:
I think you should throw `NotImplementedException` or atleast return empty array.